content/desktop/synchronized-file-sharing.md

```diff
@@ -1,73 +1,55 @@
 - Regularly encounter performance limitations.
 - Don't want to worry about file ownership or spend time resolving conflicting file-ownership information when modifying multiple containers.
 
-## How does Synchronized file sharing work?
+## How does Synchronized File Shares work?
 
 A Synchronized file share behaves just like a virtual file share, but takes advantage of Mutagen's high-performance, low-latency code synchronization engine to create a synchronized cache of the host files on an ext4 filesystem within the Docker Desktop VM. If you make filesystem changes on the host or in the VM’s containers, it propagates via bidirectional synchronization.
 
-After creating a file share instance, any container using a bind mount that points to a location on the host filesystem matching the specified synchronized file share location, utilizes the Synchronized file sharing feature. Bind mounts that don't satisfy this condition are passed to the normal virtual filesystem bind-mounting mechanism, for example VirtioFS or gRPC-FUSE.
+After creating a file share instance, any container using a bind mount that points to a location on the host filesystem matching the specified synchronized file share location, utilizes the Synchronized File Shares feature. Bind mounts that don't satisfy this condition are passed to the normal virtual filesystem [bind-mounting mechanism](../storage/bind-mounts.md), for example VirtioFS or gRPC-FUSE.
 
 > **Important**
 >
-> Synchronized file sharing isn't available on WSL or when using Windows containers. 
+> Synchronized File Shares isn't available on WSL or when using Windows containers. 
 { .important }
 
-## How to enable Synchronized file sharing
-
-Before you get started with Synchronized file sharing, you need to:
-
-- Download and install [Docker Desktop version 4.25](release-notes.md).
-- Have a Pro, Team, or Business subscription.
+## Create a file share instance 
 
-To enable Synchronized file sharing:
+To create a file share instance:
 1. Sign in to Docker Desktop.
 2. In **Settings**, navigate to the **File sharing** tab within the **Resources** section. 
-3. On your keyboard, press and hold the `Shift` key along with the `Command` key (Mac) or `Windows` key (Windows), then click anywhere within the description text to reveal a pop-up.
-4. Select **Restart**. The Docker Dashboard closes.
-
-When you re-open the Docker Dashboard, Synchronized file sharing is enabled and ready for you to create your first file share instance.
-
-## Create a file share instance 
-
-To create a synchronized file share:
-1. In **Settings**, navigate to the **File sharing** tab within the **Resources** section. 
-2. In the **Synchronized file shares** section, select the **plus** icon.
-3. Select a host folder to share. The synchronized file share should initialize and be usable.
+3. In the **Synchronized File Shares** section, select the **plus** icon.
```

Comment:
We can now replace "select the **plus** icon" with "click the 'Create share' button"